.github/workflows/artifacts-build.yml

```diff
@@ -572,70 +572,6 @@ jobs:
           git checkout empty
           gsutil -m cp -r benchmarks/binsize/* gs://${{ env.GCP_MAIN_BUCKET_ID }}/gha/benchmarks/binsize
 
-  # Data size benchmark: track size of provider/datagen/tests/data/testdata.postcard (total data size).
-  bench-datasize:
-    # This is too expensive to run on every push, so only run it on main.
-    # When running on a PR, comment this out and set the BASELINE variable below to the baseline commit.
-    if: github.ref == 'refs/heads/main' && github.repository == 'unicode-org/icu4x'
-    concurrency:
-      # Allow one run at a time to include the previous run's results
-      group: bench-datasize
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-
-    # GCP Boilerplate for jobs in main repository
-    - id: gcp-auth
-      name: "Authenticate to Google Cloud with Workload Identity Provider"
-      uses: google-github-actions/auth@v1
-      with:
-        workload_identity_provider: "projects/66042061814/locations/global/workloadIdentityPools/icu4x-gha-pool1/providers/icu4x-gha-provider1"
-        service_account: "icu4x-main@dev-infra-273822.iam.gserviceaccount.com"
-    - name: "Set up Google Cloud SDK"
-      uses: google-github-actions/setup-gcloud@v1
-
-    - name: Install rustfmt
-      run: |
-        rustup component add rustfmt
-
-    - name: Setup output data directory
-      run: |
-        mkdir -p benchmarks/datasize
-
-    - name: Generate testdata
-      run: |
-        cargo run --bin make-testdata-legacy --manifest-path tools/testdata-scripts/Cargo.toml
-
-    - name: Measure size of selected data package provider/testdata/data/testdata.postcard
-      run: |
-        cargo run --package icu_benchmark_binsize  -- provider/testdata/data/testdata.postcard file | tee benchmarks/datasize/output.txt
```

Comment:
Thought: It might be nice to run this benchmark with a postcard file generated from plain datagen, but this benchmark is not very useful because no one should be including the whole postcard file.

Reply:
We should have per-locale-per-key stats somehow, to catch things like the dictionary snafu